README.md

```diff
@@ -59,6 +59,35 @@ If you are building a plugin that requires the utilities provided by `utils.nvim
 },
 ```
 
+#### ⚙️ Configuration
+
+`utils.nvim` can optionally be configured by specifying `opts` with Lazy or alternatively with the `setup` function like so:
+
+```lua
+-- Lazy:
+{
+    'yourname/plugin.nvim',
+    dependencies = {
+        {
+            '2kabhishek/utils.nvim',
+            opts = {
+                -- provider for results from `open_dir`
+                -- can be either 'telescope' or 'fzf_lua'
+                -- defaults to 'telescope'
+                open_dir_provider = "telescope"
```

Comment:
lets change the name to `fuzzy_provider` it will give us some scope to expand this capability in future

Reply:
Sounds good - renamed :)